<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="BCMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buli Commander</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QMenuBar" name="mainMenuBar">
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="actionFolderNew"/>
      <addaction name="separator"/>
      <addaction name="actionFileOpen"/>
      <addaction name="actionFileOpenCloseBC"/>
      <addaction name="actionFileOpenAsNewDocument"/>
      <addaction name="actionFileOpenAsNewDocumentCloseBC"/>
      <addaction name="separator"/>
      <addaction name="actionFileCopyToOtherPanel"/>
      <addaction name="actionFileMoveToOtherPanel"/>
      <addaction name="separator"/>
      <addaction name="actionFileDelete"/>
      <addaction name="actionFileRename"/>
      <addaction name="separator"/>
      <addaction name="actionQuit"/>
     </widget>
     <widget class="QMenu" name="menuEdit">
      <property name="title">
       <string>&amp;Edit</string>
      </property>
      <addaction name="actionSelectAll"/>
      <addaction name="actionSelectNone"/>
      <addaction name="actionSelectInvert"/>
      <addaction name="separator"/>
      <addaction name="actionSelectRegEx"/>
     </widget>
     <widget class="QMenu" name="menuGo">
      <property name="title">
       <string>&amp;Go</string>
      </property>
      <widget class="QMenu" name="menuGoSavedViews">
       <property name="toolTip">
        <string>Saved views</string>
       </property>
       <property name="statusTip">
        <string>Saved views</string>
       </property>
       <property name="title">
        <string>&amp;Saved views</string>
       </property>
       <addaction name="actionGoSavedViews_NoSavedViewsYet"/>
      </widget>
      <widget class="QMenu" name="menuGoHistory">
       <property name="title">
        <string>&amp;History</string>
       </property>
       <property name="toolTipsVisible">
        <bool>true</bool>
       </property>
       <addaction name="actionGoHistory_clearHistory"/>
      </widget>
      <widget class="QMenu" name="menuGoFavorites">
       <property name="toolTip">
        <string>Favorites places</string>
       </property>
       <property name="statusTip">
        <string>Favorites places</string>
       </property>
       <property name="title">
        <string>&amp;Favorites</string>
       </property>
       <property name="toolTipsVisible">
        <bool>true</bool>
       </property>
       <addaction name="actionGoFavorites_NoFavoritesYetDefined"/>
      </widget>
      <addaction name="actionGoUp"/>
      <addaction name="actionGoBack"/>
      <addaction name="menuGoHistory"/>
      <addaction name="separator"/>
      <addaction name="menuGoFavorites"/>
      <addaction name="menuGoSavedViews"/>
     </widget>
     <widget class="QMenu" name="menuView">
      <property name="title">
       <string>&amp;View</string>
      </property>
      <addaction name="actionViewDetailled"/>
      <addaction name="actionViewSmallIcon"/>
      <addaction name="actionViewMediumIcon"/>
      <addaction name="actionViewLargeIcon"/>
      <addaction name="separator"/>
      <addaction name="actionViewShowImageFileOnly"/>
      <addaction name="actionViewShowBackupFiles"/>
      <addaction name="actionViewShowHiddenFiles"/>
      <addaction name="separator"/>
      <addaction name="actionViewDisplaySecondaryPanel"/>
      <addaction name="separator"/>
      <addaction name="actionViewSwapPanels"/>
     </widget>
     <widget class="QMenu" name="menuTools">
      <property name="title">
       <string>&amp;Tools</string>
      </property>
      <widget class="QMenu" name="menuBuliCommanderScripts">
       <property name="title">
        <string>Buli &amp;Commander scripts</string>
       </property>
       <addaction name="actionBCSManageScripts"/>
       <addaction name="separator"/>
       <addaction name="actionBCS_NoScriptYetDefined"/>
      </widget>
      <addaction name="actionToolsSearch"/>
      <addaction name="actionToolsStatistics"/>
      <addaction name="actionToolsSynchronizePanels"/>
      <addaction name="separator"/>
      <addaction name="actionConsole"/>
      <addaction name="menuBuliCommanderScripts"/>
     </widget>
     <widget class="QMenu" name="menuSettings">
      <property name="title">
       <string>&amp;Settings</string>
      </property>
      <addaction name="actionSettingsPreferences"/>
      <addaction name="separator"/>
      <addaction name="actionSettingsSaveSessionOnExit"/>
      <addaction name="actionSettingsResetSessionToDefault"/>
     </widget>
     <widget class="QMenu" name="menuHelp">
      <property name="title">
       <string>&amp;Help</string>
      </property>
      <addaction name="actionHelpAboutBC"/>
     </widget>
     <addaction name="menuFile"/>
     <addaction name="menuEdit"/>
     <addaction name="menuGo"/>
     <addaction name="menuView"/>
     <addaction name="menuTools"/>
     <addaction name="menuSettings"/>
     <addaction name="menuHelp"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMainView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitterMainView">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="BCMainViewTab" name="mainViewTab0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="BCMainViewTab" name="mainViewTab1">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStatusBar" name="mainStatusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeGripEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionFolderNew">
   <property name="text">
    <string>&amp;New folder...</string>
   </property>
   <property name="toolTip">
    <string>Create a new folder in current directory</string>
   </property>
   <property name="statusTip">
    <string>Create a new folder in current directory</string>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="text">
    <string>&amp;Open file</string>
   </property>
   <property name="toolTip">
    <string>Open the selected file in Krita and keep Buli Commander opened</string>
   </property>
   <property name="statusTip">
    <string>Open the selected file in Krita and keep Buli Commander opened</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileCopyToOtherPanel">
   <property name="text">
    <string>&amp;Copy to other panel</string>
   </property>
   <property name="toolTip">
    <string>Copy selected files to other panel location</string>
   </property>
   <property name="statusTip">
    <string>Copy selected files to other panel location</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFileMoveToOtherPanel">
   <property name="text">
    <string>&amp;Move to other panel</string>
   </property>
   <property name="toolTip">
    <string>Move selected files to other panel location</string>
   </property>
   <property name="statusTip">
    <string>Move selected files to other panel location</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionFileDelete">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete selected files</string>
   </property>
   <property name="statusTip">
    <string>Delete selected files</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionFileRename">
   <property name="text">
    <string>&amp;Rename...</string>
   </property>
   <property name="toolTip">
    <string>Rename selected files</string>
   </property>
   <property name="statusTip">
    <string>Rename selected files</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionFileToArchive">
   <property name="text">
    <string>&amp;Archive...</string>
   </property>
   <property name="toolTip">
    <string>Pack selected selected files in an archive (tar.gz, zip)</string>
   </property>
   <property name="statusTip">
    <string>Pack selected selected files in an archive (tar.gz, zip)</string>
   </property>
   <property name="whatsThis">
    <string>Archive selected files (tar.gz, zip)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Buli Commander</string>
   </property>
   <property name="statusTip">
    <string>Quit Buli Commander</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFileOpenCloseBC">
   <property name="text">
    <string>Open &amp;file and close Buli Commander</string>
   </property>
   <property name="toolTip">
    <string>Open selected file in Krita and close Buli Commander</string>
   </property>
   <property name="statusTip">
    <string>Open selected file in Krita and close Buli Commander</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all files</string>
   </property>
   <property name="statusTip">
    <string>Select all files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="text">
    <string>Select &amp;none</string>
   </property>
   <property name="toolTip">
    <string>Unselect all files</string>
   </property>
   <property name="statusTip">
    <string>Unselect all files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionSelectInvert">
   <property name="text">
    <string>&amp;Invert selection</string>
   </property>
   <property name="toolTip">
    <string>Invert selected files</string>
   </property>
   <property name="statusTip">
    <string>Invert selected files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionGoUp">
   <property name="text">
    <string>&amp;Up</string>
   </property>
   <property name="toolTip">
    <string>Climb up the folder tree</string>
   </property>
   <property name="statusTip">
    <string>Climb up the folder tree</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionGoBack">
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="toolTip">
    <string>Go back to previous folder</string>
   </property>
   <property name="statusTip">
    <string>Go back to previous folder</string>
   </property>
  </action>
  <action name="actionGoFavorites_NoFavoritesYetDefined">
   <property name="text">
    <string>(&amp;no favorites yet defined)</string>
   </property>
  </action>
  <action name="actionGoSavedViews_NoSavedViewsYet">
   <property name="text">
    <string>(&amp;no saved views yet)</string>
   </property>
  </action>
  <action name="actionViewDetailled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Detailled view</string>
   </property>
   <property name="toolTip">
    <string>A view with detailled file informations</string>
   </property>
   <property name="statusTip">
    <string>A view with detailled file informations</string>
   </property>
  </action>
  <action name="actionViewSmallIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Small icon view</string>
   </property>
   <property name="toolTip">
    <string>A small icon view with minimum informations</string>
   </property>
   <property name="statusTip">
    <string>A small icon view with minimum informations</string>
   </property>
  </action>
  <action name="actionViewMediumIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Medium icon view</string>
   </property>
   <property name="toolTip">
    <string>A medium icon view with minimum informations</string>
   </property>
   <property name="statusTip">
    <string>A medium icon view with minimum informations</string>
   </property>
  </action>
  <action name="actionViewLargeIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Large icon view</string>
   </property>
   <property name="toolTip">
    <string>A large icon view with minimum informations</string>
   </property>
   <property name="statusTip">
    <string>A large icon view with minimum informations</string>
   </property>
  </action>
  <action name="actionViewShowImageFileOnly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Krita managed image files only</string>
   </property>
   <property name="toolTip">
    <string>Only image files recognized by Krita will be shown in list</string>
   </property>
   <property name="statusTip">
    <string>Only image files recognized by Krita will be shown in list</string>
   </property>
  </action>
  <action name="actionViewShowHiddenFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;hidden files</string>
   </property>
   <property name="toolTip">
    <string>Show hidden files in list</string>
   </property>
   <property name="statusTip">
    <string>Show hidden files in list</string>
   </property>
  </action>
  <action name="actionViewDisplaySecondaryPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display secondary panel (&amp;right side)</string>
   </property>
   <property name="toolTip">
    <string>Display secondary panel (right side)</string>
   </property>
   <property name="statusTip">
    <string>Display secondary panel (right side)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionViewDisplayRightSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;right sidebar</string>
   </property>
   <property name="toolTip">
    <string>Display sidebar from right panel</string>
   </property>
   <property name="statusTip">
    <string>Display sidebar from right panel</string>
   </property>
  </action>
  <action name="actionViewSwapPanels">
   <property name="text">
    <string>Swap &amp;panels</string>
   </property>
   <property name="toolTip">
    <string>Swap left and right panels</string>
   </property>
   <property name="statusTip">
    <string>Swap left and right panels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionViewShowBackupFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;backup files</string>
   </property>
   <property name="toolTip">
    <string>Show backup files in list</string>
   </property>
   <property name="statusTip">
    <string>Show backup files in list</string>
   </property>
  </action>
  <action name="actionToolsSearch">
   <property name="text">
    <string>&amp;Search...</string>
   </property>
   <property name="toolTip">
    <string>Open advanced search window</string>
   </property>
   <property name="statusTip">
    <string>Open advanced search window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToolsStatistics">
   <property name="text">
    <string>Stat&amp;istics...</string>
   </property>
   <property name="toolTip">
    <string>Calculate statistics for current directory</string>
   </property>
   <property name="statusTip">
    <string>Calculate statistics for current directory</string>
   </property>
  </action>
  <action name="actionToolsSynchronizePanels">
   <property name="text">
    <string>Synchronize &amp;panels...</string>
   </property>
   <property name="toolTip">
    <string>Synchronize files between panels</string>
   </property>
   <property name="statusTip">
    <string>Synchronize files between panels</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>&amp;Buli Commander terminal</string>
   </property>
   <property name="toolTip">
    <string>Open Buli Commander terminal</string>
   </property>
   <property name="statusTip">
    <string>Open Buli Commander terminal</string>
   </property>
  </action>
  <action name="actionSettingsPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="toolTip">
    <string>Open Buli Commander settings window</string>
   </property>
   <property name="statusTip">
    <string>Open Buli Commander settings window</string>
   </property>
  </action>
  <action name="actionHelpAboutBC">
   <property name="text">
    <string>&amp;About Buli Commander...</string>
   </property>
   <property name="iconText">
    <string>About Buli Commander...</string>
   </property>
   <property name="toolTip">
    <string>Open About Buli Commander window</string>
   </property>
   <property name="statusTip">
    <string>Open About Buli Commander window</string>
   </property>
  </action>
  <action name="actionSelectRegEx">
   <property name="text">
    <string>&amp;RegEx select...</string>
   </property>
   <property name="toolTip">
    <string>Select files matching given regular expression</string>
   </property>
   <property name="statusTip">
    <string>Select files matching given regular expression</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionGoHistory_clearHistory">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear history</string>
   </property>
  </action>
  <action name="actionBCSManageScripts">
   <property name="text">
    <string>&amp;Manage scripts...</string>
   </property>
  </action>
  <action name="actionBCS_NoScriptYetDefined">
   <property name="text">
    <string>(&amp;no script yet defined)</string>
   </property>
  </action>
  <action name="actionFileOpenAsNewDocument">
   <property name="text">
    <string>O&amp;pen file as new document</string>
   </property>
   <property name="toolTip">
    <string>Open the selected file in Krita as a new document and keep Buli Commander opened</string>
   </property>
   <property name="statusTip">
    <string>Open the selected file in Krita as a new document and keep Buli Commander opened</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFileOpenAsNewDocumentCloseBC">
   <property name="text">
    <string>Open file as new document and close &amp;Buli Commander</string>
   </property>
   <property name="toolTip">
    <string>Open selected file in Krita as new document and close Buli Commander</string>
   </property>
   <property name="statusTip">
    <string>Open selected file in Krita as new document and close Buli Commander</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionSettingsResetSessionToDefault">
   <property name="text">
    <string>&amp;Reset session to default</string>
   </property>
   <property name="statusTip">
    <string>Reset session to default</string>
   </property>
  </action>
  <action name="actionSettingsSaveSessionOnExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save session on exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BCMainWindow</class>
   <extends>QDialog</extends>
   <header>bulicommander.bcmainwindow</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BCMainViewTab</class>
   <extends>QFrame</extends>
   <header>bulicommander.bcmainviewtab</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
